### PR TITLE
Update Sendgrind Email Addresses + Linting

### PR DESCRIPTION
**Pull request by hongyi-chen**
## Description

Link to ticket: https://www.notion.so/uwblueprintexecs/Update-Sendgrid-email-address-da9de410479d4bdbbd8c7e9d5d9d69b7

## Testing

Please test the following cases & verify that the email that it is sent from is: `info@raisingtheroof.org`

- User creates an account & receives verification email
- User creates a team and send invite emails to others
- User resets password
- User attempts to reset password (but their email address is not linked with an account)

If you're going to be testing the reply-to emails, please change the email from `skovacev@raisingtheroof.org` to one of your own email addresses (i.e. cmd-f for all instances of this and replace it locally, but don't commit changes). You can test the reply-to by actually trying to reply to any of the emails that are sent to you.

### CODE REVIEW

File: src/resolvers/auth.js [hunk 1]
```diff
@@ -18,17 +18,19 @@ const { CLIENT_URL } = require('../config/config');
 const createResetPasswordEmail = resetToken => {
   const resetPasswordUrl = `${CLIENT_URL}/resetPassword/${resetToken}`;
   return {
-    from: 'kevinzhang@uwblueprint.org',
+    from: 'info@raisingtheroof.org',
     subject: `Raising the Roof Password Reset Attempt`,
-    html: `Reset your password <a href="${resetPasswordUrl}"> here </a>`,
```
Comment (hongyi-chen) [inline src/resolvers/auth.js:23]: We should probably add more context to these copies since they're pretty vague and can seem sketchy?

Comment (jzxhuang) [reply src/resolvers/auth.js:23]: yeah, we should create an email template with a footer and all, ugh - gonna track a notion ticket https://www.notion.so/uwblueprintexecs/cc098375fd224e09b6efd05f453b1c5c?v=e4bd24ffa7aa470ba4a7f6b989eae603&p=fb8dee85d3454e24a484a994ab3f5b11

**Review by jzxhuang — COMMENTED** [review]
lgtm. didnt' test functionality but I'm sure it works, unless Sendgrid dies for some reason

Thoughts on reading these in as env variables — or at least as a global constant somewhere?

File: src/services/sendEmail.js [hunk 2]
```diff
@@ -15,8 +13,9 @@ const createVerificationEmail = hash => {
   const inviteUrl = `${CLIENT_URL}/verify/${hash}`;
 
   return {
-    from: 'hongyichen@uwblueprint.org',
+    from: 'info@raisingtheroof.org',
     subject: `Verify your email for Raising the Roof's Toque Campaign`,
+    replyTo: 'skovacev@raisingtheroof.org',
     html: `Thank you for signing up! Please verify your email with this link: <a href="${inviteUrl}">${inviteUrl}</a>`,
```
Comment (cynding) [inline src/services/sendEmail.js]: Not sure if we want to hyperlink "here" like we do with resetpassword or vice versa, just to keep user experience the same :)

i.e. Thank you for signing up! Please verify your email <a href="${inviteUrl}"> here </a>

**Review by cynding — APPROVED** [review]
Looks good to me!! Tested out email with creating a new user - the other changes look really similar, so should be good

File: src/services/sendEmail.js (hunk 2 above)
Comment (hongyi-chen) [reply src/services/sendEmail.js]: fixed in b3bfe6c

**Pull request merged**